packages/cozy-harvest-lib/src/components/FlowProvider.jsx

```diff
@@ -141,7 +141,8 @@ export class FlowProvider extends Component {
     // created, this is why a trigger like object is created here
     const triggerLike = {
       message: {
-        account: flow?.account?._id
+        account: flow?.account?._id,
+        konnector: flow?.konnector?.slug
```

Comment:
What is the real trigger.message? Should we add others keys?

Reply:
This is the real trigger message keys : account and konnector